src/dos/program_serial.cpp

```diff
@@ -167,27 +167,27 @@ void SERIAL::AddMessages() {
 	        "Manages the serial ports.\n"
 	        "\n"
 	        "Usage:\n"
-	        "  [color=green]serial[reset] [color=white][PORT#][reset]                List all or specified serial ports.\n"
+	        "  [color=green]serial[reset] [color=white][PORT#][reset]                List all or specified ([color=white]1[reset], [color=white]2[reset], [color=white]3[reset], or [color=white]4[reset]) ports.\n"
 	        "  [color=green]serial[reset] [color=white]PORT#[reset] [color=cyan]TYPE[reset] [settings]  Set the specified port to the given type.\n"
```

Comment:
"List all or specified serial ports."
"Set the specified port to the given type."

These weren't particularly good descriptions  in the first place. For example, you don't set the port itself to a nullmodem; you emulate a nullmodem device plugged into the given port.

Given @kcgen's suggestion, if you change the the top-level `TYPE` to `DEVICE`, these descriptions would make things a lot clearer:

"List settings of all or specified serial ports."
"Set the emulated device for the given port, or disable the port."

Reply:
Changed top-level TYPE to DEVICE. "Set port settings" looks strange to me - I've changed to "Attach specified device to the given port" (nothing longer will fit, unfortunately). "List settings" is also too long.

Reply:
> "Attach specified device to the given port"

Yeah I guess now with `DEVICE` we introduced a discrepancy, because the "type" of the port can also be `DISABLED`, which is clearly not a device...

I think `DISABLED` as a device "type" sounds pretty silly... but `NONE` could work. How about that? Then your description "Attach specified device to the given port" would make sense (attach device, or set it to `NONE` if no device).